lib/src/layout_element.dart

```diff
@@ -251,11 +251,74 @@ TableStyleElement parseTableDefinitionElement(
   }
 }
 
+class DetailsContentElement extends LayoutElement {
```

Comment:
@erickok I will need your comment here on whether or not the summary/detail tags should be LayoutElements. I'm not intimately familiar with what super classes should be used, but I just want to double check

Reply:
I originally tried to make them `InteractableElement` but then Eric changed a lot of stuff with that so links would render correctly, and it broke how I was trying to render these tags. That left me with `ReplacedElement` or `LayoutElement` so I just thought it would be better as `LayoutElement` because it's not really "replacing" anything per-se, just "laying out" the HTML the expected way.